package.json

```diff
@@ -28,32 +28,32 @@
   "devDependencies": {
     "@istanbuljs/nyc-config-typescript": "^1.0.2",
     "@microsoft/api-extractor": "^7.38.4",
```

Comment:
Following up on Bailey's comment these are all fine and desirable to update

```txt
 @microsoft/api-extractor          ^7.38.4  →  ^7.40.5
 @typescript-eslint/eslint-plugin   ^7.0.1  →   ^7.0.2
 @typescript-eslint/parser          ^7.0.1  →   ^7.0.2
 rollup                            ^4.11.0  →  ^4.12.0
 ts-node                           ^10.9.1  →  ^10.9.2
 tsd                               ^0.28.1  →  ^0.30.5
```

Reply:
Looks like there was a minor version release over the weekend for these dependencies, since I originally updated them, too. Just re-updated them!
```
 @typescript-eslint/eslint-plugin   ^7.0.1  →   ^7.0.2
 @typescript-eslint/parser          ^7.0.1  →   ^7.0.2
 rollup                            ^4.11.0  →  ^4.12.0
```

As for ts changes, the parent ticket explicitly calls out "Updating TS is NOT part of this ticket, file a separate ticket if an update is available and desirable." I'll ping the team asking if this update is desirable.

Reply:
Per the [slack clarification ](https://mongodb.slack.com/archives/GGWBN4ZNK/p1708534618400239), I've also updated the following.
```
 ts-node                           ^10.9.1  →  ^10.9.2
 tsd                               ^0.28.1  →  ^0.30.5
```